src/Misc/layoutbin/en-US/strings.json

```diff
@@ -225,6 +226,7 @@
   "ConnectingToServer": "Connecting to server ...",
   "ConnectSectionHeader": "Connect",
   "ConnectToServer": "Connecting to the server.",
+  "ContainerWindowsVersionRequirement": "Container feature requires Windows server 1803 or higher. Please reference documentation (https://go.microsoft.com/fwlink/?linkid=875268)",
```

Comment:
is `Windows Server` the product name ? i.e. do you need to capitalize the S in Server?

Reply:
i think so, i will change it to Windows Server